src/ZoneWriting/ZoneWriting.cpp

```diff
@@ -29,9 +34,15 @@ bool ZoneWriting::WriteZone(std::ostream& stream, Zone* zone)
 
     if (zoneWriter == nullptr)
     {
-        printf("Could not create ZoneWriter for zone '%s'.\n", zone->m_name.c_str());
+        std::cerr << std::format("Could not create ZoneWriter for zone \"{}\".\n", zone->m_name);
         return false;
     }
 
-    return zoneWriter->WriteZone(stream);
+    const auto result = zoneWriter->WriteZone(stream);
+
+    const auto end = std::chrono::high_resolution_clock::now();
+
+    std::cout << std::format("Writing zone \"{}\" took {} msec.\n", zone->m_name, std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count());
```

Comment:
Oh also the other code refers to the time unit "milliseconds" as `ms`, not `msec`.
Its not that this is necessarly worse, but would be nice to be consistent in the codebase.